xdrip/View Controllers/Root View Controller/RootViewController.swift

```diff
@@ -1412,31 +1412,44 @@ final class RootViewController: UIViewController {
     private func createAndPresentSensorButtonActionSheet() {
         
         // initialize list of actions
-        var listOfActions = [String : ((UIAlertAction) -> Void)]()
+        var listOfActions = [UIAlertAction]()
```

Comment:
I kept the same approach as was already here but it might be better to instantiate the UIAlertController here and just use addAction everywhere we're adding actions.  I can make that change.